component/build.gradle

```diff
@@ -0,0 +1,24 @@
+ext.moduleName = 'com.synopsys.integration.alert.component'
+
+repositories {
+    // Need this repository to resolve the spring-security-saml2-core dependencies
+    maven {
+        url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
+    }
+}
+
+dependencies {
+    implementation project(':alert-common')
+    implementation project(':alert-database')
+    implementation project(':channel')
+    implementation project(":workflow")
+
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml'
```

Comment:
Could we make this an API dependency of channel instead of duplicating it here?

Reply:
Went through all of the build.gradle files and updated the configurations for all the libraries.